providers/azure/aci.go

```diff
@@ -193,6 +197,37 @@ func (p *ACIProvider) GetPod(namespace, name string) (*v1.Pod, error) {
 	return containerGroupToPod(cg)
 }
 
+// GetPodLogs returns the logs of a pod by name that is running inside ACI.
+func (p *ACIProvider) GetContainerLogs(namespace, podName, containerName string) (string, error) {
+	logContent := ""
+	cg, err := p.aciClient.GetContainerGroup(p.resourceGroup, fmt.Sprintf("%s-%s", namespace, podName))
+	if err != nil {
+		// Trap error for 404 and return gracefully
+		if strings.Contains(err.Error(), "ResourceNotFound") {
```

Comment:
We might want some shared values here so we don't have hard coded string matches. For example the net/http package has a bunch of constants defined https://golang.org/pkg/net/http/#pkg-constants so you can just do Status == http.StatusNotFound

Reply:
done